trailmet/datasets/classification/tcga.py

```diff
@@ -123,23 +123,23 @@ def __getitem__(self, idx):
             img = Image.open(image_path).convert('RGB')
             img = self.transform(img)
             class_name = self.labels_dict[path]
-            return img, torch.tensor([self.class_dict[class_name]])
+            return img, torch.tensor([self.class_dict[class_name]]).item()
         elif self._mode == 'val':
             path = self.Val_List[idx]
             image_path = os.path.join(self.data_path, self.labels_dict[path],
                                       path + '.png')
             img = Image.open(image_path).convert('RGB')
-            img = self.target_transform(img)
+            img = self.transform(img)
```

Comment:
Can you please use `target_transforms` itself for `val` and `test`? And also add `if` statement where if transforms are not None, then only apply them.